src/Framework/Dumper.php

```diff
@@ -307,7 +307,7 @@ public static function dumpException($e)
 			. (isset($stored) ? 'diff ' . Helpers::escapeArg($stored[0]) . ' ' . Helpers::escapeArg($stored[1]) . "\n\n" : '');
 
 		foreach ($trace as $item) {
-			$item += array('file' => NULL, 'class' => NULL, 'type' => NULL, 'function' => NULL);
+			$item += ['file' => NULL, 'class' => NULL, 'type' => NULL, 'function' => NULL];
 			if ($e instanceof AssertException && $item['file'] === __DIR__ . DIRECTORY_SEPARATOR . 'Assert.php') {
 				continue;
 			}
```

Comment:
The line bellow can be simplified to `&& strpos($tmp = file($item['file'])[$item['line'] - 1], "::$item[function](") ? $tmp : NULL;`